client/src/main/scala/org/apache/celeborn/client/commit/ReducePartitionCommitHandler.scala

```diff
@@ -82,6 +83,13 @@ class ReducePartitionCommitHandler(
   private val rpcCacheConcurrencyLevel = conf.clientRpcCacheConcurrencyLevel
   private val rpcCacheExpireTime = conf.clientRpcCacheExpireTime
 
+  private val shuffleIntegrityCheckEnabled = conf.clientShuffleIntegrityCheckEnabled
+  // partitionId-shuffleId -> number of mappers that have written to this reducer (partition + shuffle)
+  private val commitMetadataForReducer =
```

Comment:
Should clean `commitMetadataForReducer/aqePartitionCompletenessValidator` when the shuffle expired

Reply:
Ack. Removing in `removeExpiredShuffle()`